index.html

```diff
@@ -8,6 +8,11 @@
     <title>Password generator</title>
   </head>
   <body>
+    <main>
+      <h1>Password Generator</h1>
+      <h2>Click on the button to get started!</h2>
```

Comment:
```suggestion
      <h2>Click the button to get started!</h2>
```

I feel this sounds better